src/app/layout/header/header.component.ts

```diff
@@ -191,7 +191,7 @@ export class HeaderComponent implements OnInit, OnDestroy {
         // /namespace/space/create instead of /namespace/space/create/pipelines
         // as the 'Create' page matches to the 'Codebases' page
         let subMenus = (n.menus || []).slice().reverse();
-        if (subMenus) {
+        if (subMenus.length > 0) {
```

Comment:
I'd prefer a null check beforehand as well, just to be safe.

E.g. `subMenus && subMenus.length > 0`

Reply:
Sure, I've added that in.